wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -29,6 +35,7 @@ library
                        Cardano.Wallet.API.V1.Accounts
                        Cardano.Wallet.API.V1.Transactions
                        Cardano.Wallet.API.V1.Settings
+                       Cardano.Wallet.API.V1.Swagger
```

Comment:
Do you think there is any way we can avoid pulling `V1.Swagger` into the library? It really shouldn't belong there.

In theory, you should be able to simply create your `wallet-new-swagger` by depending on `Cardano.wallet.API.V1.Swagger` without moving it if you include `server` in the list of `hs-source-dirs`. have you tried that?

Reply:
Sure, but looks like cabal dislikes when executables depend on other executables

Reply:
But in this case you wouldn't depend from another executable, you would simply include the `server` source code into `hs-source-dirs`, which is completely executable-agnostic, in this case. What error you get if you try that?

Reply:
Really, it worked, thanks!